### ISSUE TITLE
Base prototype contains static `errors` field

### ISSUE THREAD

**Issue by xytis** [paymentwall/paymentwall-node#2]
This field fills up as paymentwall errors are encountered. No public methods are exposed. I think it is a security issue, since nodejs server is doomed to leak and may be ddos'ed via paymentwall pingback endpoint.

**Comment by liufanhhh** [comment]
Hi Xytis,

Thank you so much for your reminding.

We will fix this error in next version.

**Comment by xytis** [comment]
Took you a while to respond :D

**Comment by liufanhhh** [comment]
Lol, now I will maintain the paymentwall-node library. If you found any other issues, you can ping me directly or open another issue. Response will be fast :D

**Comment by liufanhhh** [comment]
Updated new structure and removed this from prototype

**Comment by jkondratowicz** [comment]
Sorry, but is this really fixed? I'm using latest version from npm (1.2.2) with NodeJS 6.10, errors are appended and leak between instances...

```javascript
const pingback1 = new Pingback({}, ip);
const pingback2 = new Pingback({}, ip);
const pingback3 = new Pingback(validData, ip);
pingback1.validate();
pingback2.validate();
pingback3.validate();
console.log(pingback3.errors);
```

Should return `[]`, returns `[ 'Wrong signature', 'Wrong signature' ]`.

**Comment by jkondratowicz** [comment]
Nevermind, fixed with 1.2.3 release.
